TELCODOCS-2043 - CNF-13766 - unnumbered BGP peering (#89663)
openshift/openshift-docs: modules/nw-metallb-bgppeer-cr.adoc

|`spec.peerAddress`
|`string`
|Specifies the IP address of the peer to contact for establishing the BGP session.
If you use this field, you cannot specify a value in the `spec.interface` field.

|`spec.interface`
|`string`
|Specifies the interface name to use when establishing a session. 
Use this field to configure unnumbered BGP peering.

> There must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).

Reviewer: Here it not in the traditional sense soley a point to point interface like a serial line or ATM. But any type in which you have layer 2 and IPV6 RA functioning.

|`spec.peerAddress`
|`string`
|Specifies the IP address of the peer to contact for establishing the BGP session.
If you use this field, you cannot specify a value in the `spec.interface` field.

|`spec.interface`
|`string`
|Specifies the interface name to use when establishing a session. 
Use this field to configure unnumbered BGP peering.

> There must be a point-to-point, layer 2 connection between the two BGP peers.

Reviewer 2: ```suggestion
You must establish a point-to-point, layer 2 connection between the two BGP peers.
```

considering the connection can be set by users

You can use unnumbered BGP peering with IPv4, IPv6, or dual-stack, but you must enable IPv6 RAs (Router Advertisements).
Each interface is limited to one BGP connection.
If you use this field, you cannot specify a value in the `spec.peerAddress` field.
The `spec.bgp.routers.neighbors.interface` field is a Technology Preview feature only. 

> For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].

Reviewer 2: ```suggestion
For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```

|`spec.peerAddress`
|`string`
|Specifies the IP address of the peer to contact for establishing the BGP session.
If you use this field, you cannot specify a value in the `spec.interface` field.

|`spec.interface`
|`string`
|Specifies the interface name to use when establishing a session. 
Use this field to configure unnumbered BGP peering.
There must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).
If you use this field, you cannot specify a value in the `spec.peerAddress` field.
The `spec.bgp.routers.neighbors.interface` field is a Technology Preview feature only. 

> Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. 

Reviewer 3: Do we really do that every time we have a TP?

I was thinking having only saying `this is TP (link)`
